arch/arm/src/tlsr82/tlsr82_pwm.c

```diff
@@ -589,16 +507,13 @@ static int pwm_cfg_check(struct tlsr82_pwmtimer_s *priv)
  * Returned Value:
  *   Zero on success; a negated errno value on failure
  *
- * Assumptions:
- *   APB1 or 2 clocking for the GPIOs has already been configured by the RCC
- *   logic at power up.
- *
  ****************************************************************************/
 
 static int pwm_setup(struct pwm_lowerhalf_s *dev)
 {
   struct tlsr82_pwmtimer_s *priv = (struct tlsr82_pwmtimer_s *)dev;
   uint32_t pincfg;
+  int ret = OK;
```

Comment:
seems like init value is not needed. Let's remove `= OK;` or change `return OK;` to `return ret;`

Reply:
Done